src/main/java/at/hannibal2/skyhanni/features/garden/GardenYawAndPitch.kt

```diff
@@ -18,10 +20,9 @@ class GardenYawAndPitch {
 
     @SubscribeEvent
     fun onRenderOverlay(event: GuiRenderEvent.GuiOverlayRenderEvent) {
-        if (!LorenzUtils.inSkyBlock) return
-        if (!config.enabled) return
```

Comment:
this line shouldnt be removed

Reply:
Yes it should.
The checks were changed. Look at the line below it.